cmd/telemetry-bench.go

```diff
@@ -144,6 +144,32 @@ func (m *plugin) GetMetricMessage() (msgs []string) {
 	return buffers
 }
 
+//GetEventMessage generate mock collectd event messages
+func (m *plugin) GetEventMessage() (msg []string) {
+	bufferSize := len(m.mtype) * len(m.typeInstance) * len(m.pluginInstance)
+	buffers := make([]string, bufferSize)
+
+	msgMax := cap(m.mtype) * cap(m.pluginInstance) * cap(m.typeInstance)
+	for msgCount := 0; msgCount < msgMax; msgCount++ {
+		var sb strings.Builder
+
+		sb.Grow(1024)
+		sb.WriteString(`[{"labels":{"alertname":"event_interface_if_octets","instance":"`)
+
+		sb.WriteString(*m.hostname)
```

Comment:
This looks weird to me; there is a hostname field in the notification structure, and I don't see it getting special treatment to become an "instance" label instead anywhere in the collectd code (though I could have missed it!). I'd expect it in a field called "hostname", but a sample message would be more convincing than my quick read-through of the collectd code.

Reply:
I believe I based that label decision on [this line](https://github.com/collectd/collectd/blob/6b0e22a545bb3e4be1684a9fd6d2b48259ba9a7f/src/utils/format_json/format_json.c#L563) in the collectd code.

Reply:
Perfect; I don't know how I missed it.